src/firefly/java/edu/caltech/ipac/firefly/server/util/QueryUtil.java

```diff
@@ -534,10 +534,10 @@ public static DataAccessException createEndUserException(String msg, String deta
     }
 
 
-    private static final int DECI_DEF_MAX_POINTS = AppProperties.getIntProperty("decimation.def.max.points", 100000);
+    private static final int DECI_DEF_MAX_POINTS = AppProperties.getIntProperty("decimation.def.max.points", 10000);
```

Comment:
I think this is the way we want it.  The number is a fallback to the property.  Setting the property will override it.